lib/shutdown.js

```
@@ -75,5 +75,9 @@ exports.handleTerminationSignals = function(app, callback) {
     };
   }
 
-  process.on('SIGINT', endIt('INT')).on('SIGTERM', endIt('TERM')).on('SIGQUIT', endIt('QUIT'));
+  try {
+    process.on('SIGINT', endIt('INT')).on('SIGTERM', endIt('TERM')).on('SIGQUIT', endIt('QUIT'));
+  } catch (noSignals) {
```

@seanmonstar - Is there any way of testing whether an environment supports signals before wrapping this with a try/catch?  I am worried that this may swallow valid exceptions thrown on *nix based systems when shutting down.